README.md

```diff
@@ -29,24 +30,10 @@ Test cases MUST be written using [Intern](https://theintern.github.io) using the
 
 To test locally in node run:
 
-`grunt test`
-
-To test against browsers with a local selenium server run:
-
-`grunt test:local`
-
-To test against BrowserStack or Sauce Labs run:
-
-`grunt test:browserstack`
-
-or
-
-`grunt test:saucelabs`
+`grunt test:node`
```

Comment:
probably best to override the `grunt test` task to run everything it needs for tests